manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -21,6 +21,7 @@ spec:
   selector:
     matchLabels:
       k8s-app: istio-cni-node
+      istio.io/rev: {{ .Values.revision | default "default" }}
```

Comment:
may not be needed given we don't support revision of istio-cni atm?

Reply:
Yep, the current (and future) CNI agent doesn't use this label for anything, and the CNI agent doesn't talk to the control plane, so this label does nothing.

Reply:
I can def remove this, it just seemed weird that the deployment has a revision label, and the pod doesn't. I think both should either have it or not have it.

Reply:
None of the `istio-cni` resources should have it, yep.